bot/cogs/ext/commands/anime.py

```diff
@@ -1,11 +1,12 @@
 """Father, forgive me for these sins."""
 
 import asyncio
-from typing import List, Union
 import discord
 import random
 import re
 
+from typing import List, Union
+
```

Comment:
You added an extra line

Reply:
import re

from typing import List, Union

from discord.ext import commands
isnt that what you wanted

Reply:
Let me spoon feed you

```py
import asyncio
import discord
import random
import re

from discord.ext import commands
from discord.ext.commands.context import Context
from discord.ext.commands.errors import BadArgument
from typing import List, Union

from constants import JESSE_ID, THEOTOWN_GUILD
from cogs.utils import http
from cogs.utils.embeds import create_embed, build_embed, error
from cogs.utils.paginators import ListPageSource, PidroidPages
from cogs.utils.parsers import truncate_string
from cogs.models.waifuListApi import MyWaifuListAPI, Waifu, WaifuSearchResult
from cogs.models.categories import RandomCategory
```

Reply:
now that just means it was incorrect before i even started

Reply:
Probable, anyway, it's good exercise to fix it right now :)